tests/ipvs_tests.py

```diff
@@ -211,6 +211,20 @@ def test_dest_port(self):
                 'Service port {1}.'.format(dest.port(), service.port())
             )
 
+    def test_dest_diff_port(self):
+        self.client.add_service('1.1.1.1', 80)
+        self.client.add_dest('1.1.1.1', 80, '2.2.2.1', 8080)
+        self.client.add_dest('1.1.1.1', 80, '2.2.2.2', 8080, weight=100)
```

Comment:
could you add a test where we have 2 destinations with the same ip and different ports.